src/lib/components/EmailDetailView.svelte

```diff
@@ -185,6 +185,63 @@
 		placement: 'bottom',
 		closeQuery: ''
 	};
+
+	const icons = {
```

Comment:
Can you please move these to their own file? It'll make this svelte easier to read